### PR TITLE
fix lldb for later android versions

### PR DESCRIPTION
**Pull request by its-pointless**
fix for #4282

### CODE REVIEW

File: packages/lldb/ProcessLauncherPosixFork.cpp.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--- source/Host/posix/ProcessLauncherPosixFork.cpp.orig	2019-11-03 00:42:25.977205716 +0000
++++ ./source/Host/posix/ProcessLauncherPosixFork.cpp	2019-11-03 00:43:11.313499292 +0000
+@@ -136,8 +136,8 @@
+ 
+   if (info.GetFlags().Test(eLaunchFlagDebug)) {
+     // Do not inherit setgid powers.
+-    if (setgid(getgid()) != 0)
+-      ExitWithError(error_fd, "setgid");
++ //   if (setgid(getgid()) != 0)
++   //   ExitWithError(error_fd, "setgid");
```
Comment (buttaface) [inline packages/lldb/ProcessLauncherPosixFork.cpp.patch]: Wouldn't it be better to put this behind a `#if __ANDROID_API__ < 28` check?

Comment (its-pointless) [reply packages/lldb/ProcessLauncherPosixFork.cpp.patch]: If this was going upstream to patch it properly i would be doing a lot of other stuff to detect android version at runtime not compile time.

Comment (its-pointless) [reply packages/lldb/ProcessLauncherPosixFork.cpp.patch]: remember most of the time a debugger is used via adb or with root so setting gid is not likely needed unless we are making an aosp rom. And even then...

Comment (buttaface) [reply packages/lldb/ProcessLauncherPosixFork.cpp.patch]: Oh right, forgot that we set the API to 24, so that wouldn't fix anything, never mind.

**Pull request merged**